src/v/config/rest_authn_endpoint.cc

```diff
@@ -44,6 +44,15 @@ std::ostream& operator<<(std::ostream& os, const rest_authn_endpoint& ep) {
     return os;
 }
 
+rest_authn_method get_authn_method(
+  const std::vector<rest_authn_endpoint>& endpoints, size_t listener_idx) {
+    if (listener_idx < 0 || listener_idx >= endpoints.size()) {
```

Comment:
> listener_idx < 0

nit. i realize this is just moving code around, but a `size_t` cannot be less than 0

Reply:
nothing to be done here unless you want to.